### ISSUE TITLE
model/lstm_attention.py

### ISSUE THREAD

**Issue by temur3559** [temur3559/UltimateCryptoBotV2#9]
import numpy as np
from tensorflow.keras.models import Model
from tensorflow.keras.layers import LSTM, Dense, Input, Attention, Concatenate

def build_model(input_shape):
    inputs = Input(shape=input_shape)
    x = LSTM(64, return_sequences=True)(inputs)
    
    attention = Attention()([x, x])
    x = Concatenate()([x, attention])
    
    x = LSTM(32)(x)
    output = Dense(1)(x)
    
    model = Model(inputs, output)
    model.compile(optimizer='adam', loss='mse')
    return model
